opentelemetry-example/src/main/scala/zio/telemetry/opentelemetry/example/http/ProxyHttpApp.scala

```diff
@@ -1,21 +1,21 @@
 package zio.telemetry.opentelemetry.example.http
 
-import io.opentelemetry.api.trace.{ SpanKind, StatusCode }
+import io.opentelemetry.api.trace.{SpanKind, StatusCode}
 import zhttp.http._
 import zio._
 import zio.json.EncoderOps
 import zio.telemetry.opentelemetry.baggage.Baggage
 import zio.telemetry.opentelemetry.baggage.propagation.BaggagePropagator
 import zio.telemetry.opentelemetry.context.OutgoingContextCarrier
 import zio.telemetry.opentelemetry.tracing.propagation.TraceContextPropagator
-import zio.telemetry.opentelemetry.tracing.{ ErrorMapper, Tracing }
+import zio.telemetry.opentelemetry.tracing.{ErrorMapper, Tracing}
 
 case class ProxyHttpApp(client: Client, tracing: Tracing, baggage: Baggage) {
 
   import tracing.aspects._
 
   private val errorMapper: ErrorMapper[Throwable] =
-    ErrorMapper[Throwable] { case _ => StatusCode.UNSET }
+    ErrorMapper[Throwable]({ case _ => StatusCode.UNSET }, Some(identity))
```

Comment:
If we apply the change I suggested in the comment below https://github.com/zio/zio-telemetry/pull/695/files#r1203978344, the definition of this one would be the same as before:
```
  private val errorMapper: ErrorMapper[Throwable] =
    ErrorMapper[Throwable] { case _ => StatusCode.UNSET }
```
Which looks much cleaner from the end user's perspective.